pages/linux/uuid.md

```diff
@@ -0,0 +1,32 @@
+# uuid
+
+> Generate and decode an Universally Unique Identifier (UUID).
+> More information: <https://manned.org/uuid.1>.
+
+- Generate a v1 UUID (time and node based):
+
+`uuid`
+
+- Generate a v4 UUID (random data based):
+
+`uuid -v {{4}}`
+
+- Generate a v3 UUID (name based) with an URL namespace:
+
+`uuid -v {{3}} ns:URL {{https://example.com}}`
+
+- Generate multiple v1 UUIDs at once by specifying a count:
+
+`uuid -n {{12}}`
+
+- Generate a v1 UUID in a single integer value (SIV) representation format:
+
+`uuid -F {{SIV}}`
```

Comment:
What do you think of:
```suggestion
- Generate a v1 UUID and specify the output format:

`uuid -F {{BIN|STR|SIV}}`
```
?

Reply:
have not encountered a pipe | here yet, but definitely resembles well known syntax

Reply:
We don't use it for actual syntax, we use it to indicate that there are multiple options.
https://sourcegraph.com/search?q=+repo:%5Egithub%5C.com/tldr-pages/tldr%24+%5C%7B%5C%7B%28%5B%5E%7B%7D%5D%2B%3F%5C%7C%29%2B%5B%5E%7B%7D%5D%2B%5C%7D%5C%7D&patternType=regexp

Reply:
Nice service that sourcegraph, thanks.